### DISCUSSION TITLE
p/9472ef44b098409a90673b353db5acf8/

### DISCUSSION THREAD

**Discussion by giscus** [rxliuli/blog#28] [Announcements]
# p/9472ef44b098409a90673b353db5acf8/

终于，过年了呢，今天是大年初一，无论如何，先祝大家农历新年快乐。[]~(￣ ▽ ￣)~* 家里今年真是少有的大团圆了，爸爸妈妈妹妹，堂哥表哥以及各种亲戚今年都回来了，也预示着可能是麻烦的一年。 年货好吧，不出意料地，身处农村的老家仍然不可免俗的购买了许多年货，许多生肉甚至很难想象可以在一个春节就能吃完。  今天赶了庙会之后还买了一些童年回忆中的零食  新年礼物好吧，m2 macbook pro 终

https://blog.rxliuli.com/p/9472ef44b098409a90673b353db5acf8/

**Comment by miaopass-future** [comment]
3w+的电脑嘛，会不会有点性能过剩

**Reply by rxliuli** [reply]
对于 ml 而言，可能还不够用（

**Reply by miaopass-future** [reply]
那确实，div主机会好很多吧，价格方面，就是不太方便、笔记本+台式机

**Reply by miaopass-future** [reply]
从前端coder转型人工智能了么

**Reply by rxliuli** [reply]
业余的，只是玩玩，跟上时代

**Reply by miaopass-future** [reply]
话说选择mac的原因是啥，因为我从未拥有过mac，用别人的就感觉触摸板很好用，做工很好，据说基本不会死机。最近也有试用的想法，但是价格方面确实贵的，同价格windows应该并不差吧。

**Reply by rxliuli** [reply]
主要是电池、命令行、小文件性能、系统可以日常不关机，其他没有什么太多的优势（某些方面，可能是劣势）

**Reply by miaopass-future** [reply]
所以你接受他的劣势就是因为电池、命令行、小文件性能、系统可以日常不关机嘛

**Reply by rxliuli** [reply]
虽然最后但并非不重要的是想要尝试变化，没有变化就是等死。windows/linux 用了超过 7 年，吾辈也想尝试一下其他系统了。

**Comment by miaopass-future** [comment]
现在没有高学历的背景在国内都不好就业

**Reply by rxliuli** [reply]
是的，你今年毕业？

**Reply by miaopass-future** [reply]
工作好几年了，只是看到文章里面写的

**Reply by miaopass-future** [reply]
打算出国留学打破僵局？

**Reply by rxliuli** [reply]
寻找变化
